### ISSUE TITLE
concept1_score and concept2_score is NaN

### ISSUE THREAD

**Issue by Hongxiii** [srkds/fcsna-qfvs#2]
`criterion=torch.nn.BCELoss()`
`loss1=criterion(concept1_score_tmp,concept1_GT_tmp.to(self.device))`
`loss2=criterion(concept2_score_tmp,concept2_GT_tmp.to(self.device))`
The loss funtion raised RuntimeError and then I found the value of 'concept1_score' and 'concept2_score' is 
```
tensor([[[nan, nan, nan,  ..., nan, nan, nan],
         [nan, nan, nan,  ..., nan, nan, nan],
         [nan, nan, nan,  ..., nan, nan, nan],
         ...,
```

**Comment by srkds** [comment]
In a seprate cell execute this forward pass with single example.

```py
seg_len = torch.tensor([171, 127, 186, 129,  73, 159, 158, 194, 136,  28, 163, 176, 135,  49,
        184, 199, 199,  21, 101, 195], dtype=torch.int32)
visual_feature = torch.randn((1,20,200, 2048), dtype=torch.float32) # B, seg, T, C
q1 = torch.randn((1,300), dtype=torch.float32) # B, C
q2 = torch.randn((1,300), dtype=torch.float32) # B, C


model = CHAN(config).to('cuda')

concept1_score,concept2_score,self_attention_wei, query_relevance_attention_wei, global_attention_wei = model(visual_feature.to('cuda'), seg_len.unsqueeze(0).to('cuda'), q1.to('cuda'), q2.to('cuda'))

concept1_score.shape

concept1_score[0][0][:10] # now you should see some random values.

```
